eng/Versions.props

```diff
@@ -25,23 +25,23 @@
     <MicrosoftVisualStudioExtensibilityTestingVersion>0.1.149-beta</MicrosoftVisualStudioExtensibilityTestingVersion>
     <!-- CodeStyleAnalyzerVersion should we updated together with version of dotnet-format in dotnet-tools.json -->
     <CodeStyleAnalyzerVersion>4.3.0-1.final</CodeStyleAnalyzerVersion>
-    <VisualStudioEditorPackagesVersion>17.5.143-preview</VisualStudioEditorPackagesVersion>
+    <VisualStudioEditorPackagesVersion>17.5.274-preview</VisualStudioEditorPackagesVersion>
```

Comment:
Can you set `VisualStudioMacEditorPackagesVersion` to `$(VisualStudioEditorPackagesVersion)` while you're here (then I wouldn't need to update https://github.com/dotnet/roslyn/pull/66545 after this merges)?

Reply:
Sure.